SPHINXsys/src/shared/geometries/complex_shape.h

```diff
@@ -56,6 +59,35 @@ namespace SPH
 	};
 
 	using DefaultShape = ComplexShape;
+
+	/**
+	 * @class AlignedBoxShape
+	 * @brief Used to describe a bounding box in which
+	 * the plane vertical to axis direction is aligned to a planar piece of a shape.
+	 */
+	class AlignedBoxShape : public TransformShape<GeometricShapeBox>
```

Comment:
The term `Aligned` is confusing, why is it having this prefix ? The class seems to provide an oriented box (since the transform is no necessarily identity)

Reply:
Please consider renaming to `OrientedBoxShape` or `BoxShape` in the next PR